app/models/user.rb

```diff
@@ -165,6 +181,22 @@ def send_account_closed_internal_notification(user_account_email)
     send_devise_notification(:account_closed_internal, user_account_email)
   end
 
+  def send_complete_registration_notification
+    send_devise_notification(:complete_registration)
+  end
+
+  def send_start_training_notification
+    send_devise_notification(:start_training)
+  end
+
+  def send_continue_training_notification(mod)
```

Comment:
Adding the missing param doc would be excellent time permitting

Reply:
[c30aa05](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/747/commits/c30aa056a3eea7ad37f7a051fea48e769a647aa1)